nautobot/dcim/tests/test_models.py

```diff
@@ -31,7 +37,62 @@
     Site,
 )
 from nautobot.extras.models import Status
+from nautobot.extras.models.customfields import CustomField
 from nautobot.tenancy.models import Tenant
+from nautobot.extras.choices import CustomFieldTypeChoices
+
+
+class InterfaceTemplateCustomFieldTestCase(TestCase):
+    def test_instantiate_model(self):
+        """
+        Check that all _custom_field_data is present and all customfields are filled with the correct default values.
+        """
+        statuses = Status.objects.get_for_model(Device)
+        site = Site.objects.create(name="Site 1", slug="site-1")
+        manufacturer = Manufacturer.objects.create(name="Acme", slug="acme")
+        device_role = DeviceRole.objects.create(name="Device Role 1", slug="device-role-1", color="ff0000")
+        custom_field_1 = CustomField.objects.create(
+            type=CustomFieldTypeChoices.TYPE_TEXT, name="field_1", default="value_1"
+        )
+        custom_field_1.content_types.set([ContentType.objects.get_for_model(Interface)])
+        custom_field_2 = CustomField.objects.create(
+            type=CustomFieldTypeChoices.TYPE_TEXT, name="field_2", default="value_2"
+        )
+        custom_field_2.content_types.set([ContentType.objects.get_for_model(Interface)])
+        custom_field_3 = CustomField.objects.create(
+            type=CustomFieldTypeChoices.TYPE_TEXT, name="field_3", default="value_3"
+        )
+        custom_field_3.content_types.set([ContentType.objects.get_for_model(Interface)])
```

Comment:
Suggested rework to slightly reduce duplicate code:

```suggestion
        custom_fields = [
            CustomField.objects.create(
                type=CustomFieldTypeChoices.TYPE_TEXT, name="field_1", default="value_1"
            ),
            CustomField.objects.create(
                type=CustomFieldTypeChoices.TYPE_TEXT, name="field_2", default="value_2"
            ),
            CustomField.objects.create(
                type=CustomFieldTypeChoices.TYPE_TEXT, name="field_3", default="value_3"
            ),
        ]
        for custom_field in custom_fields:
            custom_field.content_types.set([ContentType.objects.get_for_model(Interface)])
```

Also possibly consider using different `type` values for the different fields, just to make the test a bit broader (though it shouldn't matter in this particular case)?